src/lamplib/tasks/download.py

```diff
@@ -58,13 +58,11 @@ def fetch_and_install(self):
                         "internal volume your Mac. Verify this assumption with `diskutil list`, substituting the "
                         "appropriate volume name if required. \n\n"
```

Comment:
With the removal of the calls to diskutil, lines 57-59 are no longer relevant. Move the \n\n from line 59 to line 56